src/markdown.c

```diff
@@ -1413,6 +1413,25 @@ prefix_quote(uint8_t *data, size_t size)
 	return 0;
 }
 
+static size_t
+prefix_blockspoiler(uint8_t *data, size_t size)
+{
+        size_t i = 0;
+        if (i < size && data[i] == ' ') i++;
+        if (i < size && data[i] == ' ') i++;
+        if (i < size && data[i] == ' ') i++;
+        if (i < size && data[i] == '>') i++;
```

Comment:
By this method I think this makes the starting less than optional, and:

```

! this is something

```

May trigger a spoiler block. I think you need to marry this with the `&&` down below so that it will return zero if both `>` and `!` don't exist in tandem.